aim/cli/upgrade/utils.py

```diff
@@ -44,15 +44,15 @@ def convert_run(lrun: LegacyRun, repo: Repo, legacy_run_map, skip_failed):
         lrun.open_storage()
         if lrun.params.get(AIM_MAP_METRICS_KEYWORD):
             del lrun.params[AIM_MAP_METRICS_KEYWORD]  # set internally. no need to copy
-        run[...] = lrun.params
-        run['v2_params'] = {'run_hash': lrun.run_hash}
+        run.set_params(..., lrun.params)
```

Comment:
@mahnerak I'm open to suggestions. But since we couldn't find anything better, we decided to leave as `set_params`. At least this is consistent with the `Session` and users are familiar it. In the future we do plan to have dict-like set interface, so maybe no need to introduce new names? @gorarakelyan any thoughts?

Reply:
Agree with @alberttorosyan